src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java

```diff
@@ -69,23 +69,31 @@ public void testTokensNotNull() {
     }
 
     @Test
-    public void testWithMultilineStatements() throws Exception {
+    public void testWithMultilineStatementsOne() throws Exception {
         final String[] expected = {
             "49:21: " + getCheckMessage(MSG_KEY),
             "66:17: " + getCheckMessage(MSG_KEY),
             "74:17: " + getCheckMessage(MSG_KEY),
             "86:10: " + getCheckMessage(MSG_KEY),
             "95:28: " + getCheckMessage(MSG_KEY),
-            "140:39: " + getCheckMessage(MSG_KEY),
-            "173:46: " + getCheckMessage(MSG_KEY),
-            "184:47: " + getCheckMessage(MSG_KEY),
         };
-
         verifyWithInlineConfigParser(
-                getPath("InputOneStatementPerLineMultiline.java"),
+                getPath("InputOneStatementPerLineMultilineOne.java"),
             expected);
     }
 
+    @Test
+    public void testWithMultilineStatementsTwo() throws Exception {
+        final String[] expected = {
```

Comment:
Same here, please change the class names to be descriptive of the content, so that we know what the test is testing.

Reply:
@Vyom-Yadav done